src/test/Rules/CreateRule.test.tsx

```diff
@@ -178,4 +179,4 @@ describe('<CreateRule/>', () => {
     expect(createSpy).toHaveBeenCalledTimes(1);
```

Comment:
Wondering if we need to add a userEvent to type a value for `InitialDelaySeconds`? Either way it is still 0 as in the mock but just to be consistent with other forms?

Reply:
Sure, good idea. It's Canada Day though, stop reviewing PRs :-)

Reply:
Happy Canada Day :D i guess haha